api/nativeMessaging/README.md

```diff
@@ -0,0 +1,37 @@
+This directory contains an "MV3" example of Chromium/Chrome application that uses [Native
+Messaging](https://github.com/browserext/native-messaging) API that allows to communicate with a native application.
+
+In order for this example to work you must first install the native messaging
+host from the host directory.
+
+<h5>Install<h5>
```

Comment:
Use Markdown headings?

Reply:
I can substitute markdown for HTML if that is critical.

Reply:
It's not critical, but I am reviewing for style as well as functionality. :)

Reply:
Do you suggest 
```

Install
---
```

?